std_msgs/Header header
morefusion_ycb_video/ObjectPose[] poses
